starcounter-include.html

```diff
@@ -14,14 +14,15 @@
 
 <script>
     (function() {
-        const useShadowDOMV1 = Boolean(Element.prototype.attachShadow && Node.prototype.getRootNode);
+        const WebComponentsV0 = true; // need a way to detect WC version
```

Comment:
Why is this a problem? Can't you just check if `window.customElements !== undefined`? 

You can assume the `d-r-e.js` will be removed from Starcounter 2.3.1 (I assume we have no other option as it is not compatible with `wc.js V0`).